app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -411,7 +424,13 @@ function openFullMenu() {
         <NodeWidgetTree :ast="displayedExpression" :nodeId="nodeId" />
       </div>
     </div>
-    <GraphNodeError v-if="error" class="error" :error="error" />
+    <GraphNodeError v-if="error" class="message" :message="error" type="error" />
+    <GraphNodeError
+      v-if="warning && nodeHovered && isSelected"
```

Comment:
I'm surprised by `nodeHovered && isSelected` here, are our conditions for showing warnings that restrictive?

Reply:
That is what was requested by @AdRiley 
> Only show when node is selected and when hovering over it.

Or was that meant to mean, we show it when selected OR when hovering it?

Reply:
Yes it should be OR. I should have written that better sorry.